tests/cases/fourslash/renameInferredShorthandProperty.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path='fourslash.ts' />

```

Comment:
Please add another test:
```ts
/// <reference path='fourslash.ts' />

////interface I1 {
////    [|x|]: number;
////}
////interface I2 {
////    [|x|]: number;
////}
////function f({ [|x|] }: I1): I2 {
////    return { [|x|] };
////}

const ranges = test.ranges();
const [r0, r1, r2, r3] = ranges;

verify.renameLocations(r0, [r0, r2]);
verify.renameLocations(r1, [r1, r3]);
verify.renameLocations(r2, [r0, r2, r3]);
verify.renameLocations(r3, [r1, r2, r3]);
```
This isn't ideal since the renames will still introduce compile errors, but they did before anyway.
What we really need is a way to tell the server to change to `function f({ x: y }) { return { y }; `, but that's an issue for another day.